dll/win32/shell32/lang/de-DE.rc

```diff
@@ -1003,4 +1003,8 @@ BEGIN
     IDS_COPYTOTITLE "Wählen Sie den Ort, in den '%s' kopiert werden soll. Klicken Sie anschließend auf Kopieren."
     IDS_COPYITEMS "Elemente kopieren"
     IDS_COPYBUTTON "Kopieren"
+    IDS_MOVETOMENU "Move to &folder..."
```

Comment:
IDS_MOVETOMENU "In Ordner &verschieben..."

Reply:
OK.